packages/lib/src/components/Dropin/Dropin.tsx

```diff
@@ -11,6 +11,7 @@ class DropinElement extends UIElement<DropinElementProps> {
     public static type = 'dropin';
     protected static defaultProps = defaultProps;
     public dropinRef = null;
+    private componentFromAction = null;
```

Comment:
Is this `componentFromAction` used somewhere other than on the tests?

Reply:
No - but I don't know how else to make the `paymentAction` available to the tests since `handleAction` in the Dropin, unlike in UIElement, doesn't return the paymentAction

Reply:
Have made the creation of this prop conditional on `process.env.NODE_ENV==="test"`